### PR TITLE
Feature/more info claim

### PR DESCRIPTION
**Pull request by vonElfvin**

### CODE REVIEW

File: src/utils/DateUtil.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+export function timeAgo(pastDate: Date) {
+  const date = new Date(pastDate)
+  const today = new Date()
+  if (today.getMilliseconds() - date.getMilliseconds() > 0) {
+    throw Error('Cannot get timeAgo for date in the future')
+  }
+  return dateDifference(date, today)
+}
+
+function dateDifference(startingDate, endingDate) {
```
Comment (palmenhq) [inline src/utils/DateUtil.ts]: I _really_ don't think we should maintain code like this ourselves. There's a [date-fns function](https://date-fns.org/docs/distanceInWords) that does exactly this, but slightly better, date-fns is aleady a dependency of hope..

Comment (vonElfvin) [reply src/utils/DateUtil.ts]: Cool, didn't find any example when looking. Changed it now 👍

File: src/components/claims/claim-details/components/MemberInformation.tsx [hunk 2]
```diff
@@ -77,15 +96,34 @@ const MemberInformation: React.SFC<MemberInformationProps> = ({
...
+      <b>Personal Number:</b> {personalNumber}
+    </p>
+    <p>
+      <b>Address:</b> {address}, {postalNumber} {city}
+    </p>
+    <p>
+      <b>Failed Charge Spree:</b> {numberFailedCharges}
+    </p>
+    <p>
+      <b>Total Debt (Minimum):</b> {Math.trunc(amount)} {currency}
```
Comment (palmenhq) [inline src/components/claims/claim-details/components/MemberInformation.tsx]: Could we use [`formatMoney`](https://github.com/HedvigInsurance/hedgehog/blob/master/src/lib/intl.ts#L12) for this?

File: src/components/claims/claim-details/components/MemberInformation.tsx [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import {formatDistance} from "date-fns"
```
Comment (palmenhq) [inline src/components/claims/claim-details/components/MemberInformation.tsx]: prettier :)

**Pull request merged**